MODULE.bazel

```diff
@@ -36,7 +36,10 @@ use_repo(
     "released_com_github_jetbrains_kotlin",
 )
 
-register_toolchains("@released_rules_kotlin//kotlin/internal:default_toolchain")
+# Register bootstrap toolchain
+register_toolchains("//src/main/kotlin:bootstrap_toolchain")
+
+register_toolchains("//src/main/starlark/core/compile/...")
```

Comment:
Nit: Be explicit about toolchain registration within the rule set so we know what's being set up.

Reply:
Done.